kostyor/rest_api.py

```diff
@@ -308,4 +309,6 @@ def cluster_list():
 
 
 if __name__ == '__main__':
+    conf.parse_args(sys.argv[1:])
+    db_api.configure_session(conf.CONF.database.connection)
```

Comment:
minor: think it's better to pass the whole database config (`conf.CONF.database`).